### DISCUSSION TITLE
Deploy privateGPT for a company of 400 users

### DISCUSSION THREAD

**Discussion by WalidBabi** [imartinez/privateGPT#1617] [Q&A]
Hi, I used privateGPT and I find it helpful to deploy it on a server on-premises in my company for 400 users.
Is it possible to deploy for that many of users?

**Comment by tedraymond** [comment]
Hardware performance becomes the hurdle. Tackling this myself recently. The platform is a hit, response time is the biggest request.

Uvicorn and gradio do support ssl and authentication so security option can be enabled.
